example/package.json

```diff
@@ -11,16 +11,16 @@
   },
   "dependencies": {
     "@react-native-community/masked-view": "^0.1.10",
-    "@react-navigation/native": "^5.8.10",
-    "@react-navigation/native-stack": "5.0.4",
-    "@react-navigation/stack": "^5.12.8",
+    "@react-navigation/native": "^6.0.16",
```

Comment:
Can we blindly bump a major version of React Navigation ? Do we have to do a migration ?